5m5v-bot.js

```diff
@@ -56,7 +69,7 @@ async function getApiKey(user) {
 
           try {
             if (!isDryRun) {
-              const rettiwt = new Rettiwt({ apiKey: await getApiKey(user) });
+              const rettiwt = new Rettiwt({ apiKey: await getApiKey(user, { retry: false }) });
```

Comment:
No need to keep retrying here as we're already in a try/catch block, and we don't want the main loop to stop if only one user (language) cannot retweet.